fix: AppsProvider preventing Private Apps list from being shown on air-gapped environments (#30714)
RocketChat/Rocket.Chat: .changeset/serious-cats-fetch.md

---
'@rocket.chat/meteor': patch
---

> Fixed a problem that would prevent private apps to be shown on air-gapped environments

Author: ```suggestion
Fixed a problem that would prevent private apps from being shown on air-gapped environments
```
